src/main/java/se/bjurr/prnfb/presentation/dto/ButtonDTO.java

```diff
@@ -18,6 +18,7 @@
  private String name;
  private String projectKey;
  private String repositorySlug;
+ private String confirmation;
```

Comment:
I had a lot of difficulty with the combination of the DTO and the Velocity template for the toggle - I was unsure how to get it to translate it on the `admin.vm` side from a boolean to the "on"/"off", so I just left it as string.

If you know how to do it, happy to switch.